catalog/plugin/catalog-plugin-expirationdate/src/test/java/org/codice/ddf/catalog/plugin/expiration/TestExpirationDatePlugin.java

```diff
@@ -0,0 +1,496 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.plugin.expiration;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.data.impl.MetacardImpl;
+import ddf.catalog.operation.CreateRequest;
+import ddf.catalog.plugin.PluginExecutionException;
+import ddf.catalog.plugin.StopProcessingException;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TestExpirationDatePlugin {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TestExpirationDatePlugin.class);
+
+    private static final String DATE_FORMAT = "MM-dd-yyyy HH:mm:ss.SSS";
+
+    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormat
+            .forPattern(DATE_FORMAT);
+
+    private static final DateTime MODIFIED_DATE = DATE_TIME_FORMATTER
+            .parseDateTime("09-01-1990 00:00:00.000");
+
+    private static final DateTime ORIG_EXPIRATION_DATE = DATE_TIME_FORMATTER
+            .parseDateTime("09-10-1990 00:00:00.000");
+
+    private static final int DAYS = 10;
+
+    private ExpirationDatePlugin expirationDatePlugin;
+
+    @Mock
+    private CreateRequest mockCreateRequest;
+
+    @Before
+    public void setup() {
+        expirationDatePlugin = new ExpirationDatePlugin();
+        expirationDatePlugin.setOffsetFromModifedDate(DAYS);
+    }
+
+    @After
+    public void tearDown() {
+        expirationDatePlugin = null;
```

Comment:
Not necessary since your `@Before` method will create a new `ExpirationDatePlugin` for each test.